### PR TITLE
Remove Package-Lock.json

### PR DESCRIPTION
**Pull request by thudsonbu**
### Summary 
In order to have a better `package-lock.json` I removed

### Changes
- Removed `package-lock.json`

```javascript
const potato = 0;
```

### References
https://google.com

### CODE REVIEW

File: Tickets.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ### Home Page
 
-Index page with a breif description of the app.
+Index page with a brief description of the app.
```
Comment (thudsonbu) [inline Tickets.md:9]:
```suggestion
Index page with a brief description of the app
```

Comment (Dahvd) [inline Tickets.md:9]:
```suggestion
Index page with a brief description of the app. 💩 💩 💩 💩 
```

**Review by Dahvd — COMMENTED** [review]
💩

**Review by Dahvd — APPROVED** [review]
i approve of this deletion

**Pull request merged**